docs/gnmi_extensions.md

```diff
@@ -0,0 +1,76 @@
+# gNMI extensions on the Northbound interface
+
+gNMI is designed primarily as a device management interface, and augmenting it
+with the extra functionality to do network wide management of the configuration
+across many devices requires some extension.
+
+Some functions are far beyond what gNMI is designed for, and have been given
+their own gRPC definitons as [admin](../pkg/northbound/proto/admin.proto) and
+[diags](../pkg/northbound/proto/diags.proto).
+
+## Use of target in the NBI
+In gNMI the Path type is comprised of a set of path elements and a **target**.
+In the onos-config NBI this target represents the **name of the device** as it
```

Comment:
I would add an example and the cli command to retrieve the device names

Reply:
done